qt/python/mantidqtinterfaces/mantidqtinterfaces/FilterEvents/eventFilterGUI.py

```diff
@@ -948,36 +885,36 @@ def filterByLogValue(self):
             Logger("Filter_Events").error(error_msg)
             return
 
-        if self.ui.lineEdit_3.text() != "":
-            rel_starttime = float(self.ui.lineEdit_3.text())
+        if self.ui.leStartTime.text() != "":
+            rel_starttime = float(self.ui.leStartTime.text())
             kwargs["StartTime"] = str(rel_starttime)
 
-        if self.ui.lineEdit_4.text() != "":
-            rel_stoptime = float(self.ui.lineEdit_4.text())
+        if self.ui.leStopTime.text() != "":
+            rel_stoptime = float(self.ui.leStopTime.text())
             kwargs["StopTime"] = str(rel_stoptime)
 
-        if self.ui.lineEdit_5.text() != "":
-            minlogvalue = float(self.ui.lineEdit_5.text())
+        if self.ui.leMinimumValue.text() != "":
+            minlogvalue = float(self.ui.leMinimumValue.text())
             kwargs["MinimumLogValue"] = minlogvalue
 
-        if self.ui.lineEdit_6.text() != "":
-            maxlogvalue = float(self.ui.lineEdit_6.text())
+        if self.ui.leMaximumValue.text() != "":
+            maxlogvalue = float(self.ui.leMaximumValue.text())
             kwargs["MaximumLogValue"] = maxlogvalue
 
-        if self.ui.lineEdit_7.text() != "":
-            logvalueintv = float(self.ui.lineEdit_7.text())
+        if self.ui.leStepSize.text() != "":
+            logvalueintv = float(self.ui.leStepSize.text())
             kwargs["LogValueInterval"] = logvalueintv
         logvalchangedir = str(self.ui.comboBox_4.currentText())
         kwargs["FilterLogValueByChangingDirection"] = logvalchangedir
 
-        if self.ui.lineEdit_9.text() != "":
-            logvalueintv = float(self.ui.lineEdit_9.text())
+        if self.ui.leTimeTolerance.text() != "":
+            logvalueintv = float(self.ui.leTimeTolerance.text())
```

Comment:
Thanks for renaming these to sensible names. It was so confusing to understand what was happening before.